src/components/containers/PatientsList/__tests__/PatientsList.test.js

```diff
@@ -1,12 +1,57 @@
 import React from 'react';
-import Enzyme, { mount, shallow } from 'enzyme';
+import Enzyme, {mount, shallow} from 'enzyme';
 import Adapter from 'enzyme-adapter-react-15';
-
+import {StaticRouter} from 'react-router'
+import {Provider} from 'react-redux';
+import configureStore from 'redux-mock-store';
 import PatientsList from '../PatientsList';
+import {fetchPatientsRequest} from "../../../../ducks/feth-patients.duck";
```

Comment:
We are using a pattern where curly braces are spaced inside with the imports. Please fix all of the occurrences. `{Provide}` --> `{ Provider }`; Moreover we are using single quotes and not double quotes for the imports. Please fix that also.